src/LoginScreen.tsx

```diff
@@ -1,18 +1,36 @@
 import React from "react";
 import validateForm from "./InputValidationScript";
 import { useState } from "react";
-import e from "express";
+import { LOGIN_MUTATION } from "./services/GraphQLOperations";
+import { ApolloProvider, useMutation } from "@apollo/client";
+import { client } from "./index";
 
 function LoginScreen() {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [LoginMutation, { data, loading, error }] = useMutation(
```

Comment:
Aqui é melhor nomear como nomeamos função mesmo.

```suggestion
  const [loginMutation, { data, loading, error }] = useMutation(
```